packages/base-driver/README.md

```diff
@@ -2,15 +2,11 @@
 
 This package is part of [vscode-sqltools](https://vscode-sqltools.mteixeira.dev/?umd_source=repository&utm_medium=readme&utm_campaign=base-driver) extension.
 
-## Creating icon
+## Changelog
```

Comment:
```suggestion
## Creating icon

PNG Images
Size: 64x64px
Default Icon: Opacity 100%, no margins and paddings
Active icon: Opacity 100%, no margins and paddings, green (#00FF00) circle 24x24 bottom right
Inactive icon: Opacity 50%, no margins and paddings

## Changelog
```

Reply:
btw, can we add what was added on 0.1.10? I forgot to ask you on the previous PR, but I don't recall what change was that